app/src/main/java/com/github/se/eventradar/ui/event/EventDetails.kt

```diff
@@ -0,0 +1,150 @@
+                containerColor = Color.Transparent,
+            ),
+    ) {
+      Icon(
+          painter = painterResource(id = R.drawable.back_arrow),
+          contentDescription = "Back navigation arrow",
+          tint = MaterialTheme.colorScheme.onSurface,
+          modifier = Modifier.width(24.dp).height(24.dp).align(Alignment.CenterVertically))
+    }
+
+    Text(
+        text = eventTitle,
+        color = MaterialTheme.colorScheme.onSurface,
+        lineHeight = 20.sp,
+        fontWeight = FontWeight.Bold,
+        fontSize = titleTextSize,
+        modifier = Modifier.align(Alignment.CenterHorizontally))
+
+    Column(modifier = Modifier.padding(start = widthPadding, end = widthPadding)) {
+      Text(text = headerDescription, color = fieldTitleColor, fontSize = contentTextSize)
+      Text(text = contentDescription, color = fieldContentColor, fontSize = contentTextSize)
+    }
+
+    Row(
+        modifier =
+            Modifier.fillMaxWidth().padding(start = widthPadding, end = widthPadding).height(64.dp),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center) {
+          Column(modifier = Modifier.weight(1f)) {
+            Text(text = headerDistance, color = fieldTitleColor, fontSize = contentTextSize)
+            Text(text = contentDistance, color = fieldContentColor, fontSize = contentTextSize)
+          }
+
+          Column(modifier = Modifier.weight(1f)) {
+            Text(text = headerDate, color = fieldTitleColor, fontSize = contentTextSize)
+            Text(text = contentDate, color = fieldContentColor, fontSize = contentTextSize)
+          }
+        }
+
+    Row(
+        modifier =
+            Modifier.fillMaxWidth().padding(start = widthPadding, end = widthPadding).height(64.dp),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center) {
+          Column(modifier = Modifier.weight(1f), verticalArrangement = Arrangement.SpaceBetween) {
+            Text(text = headerCategory, color = fieldTitleColor, fontSize = contentTextSize)
+            Text(text = contentCategory, color = fieldContentColor, fontSize = contentTextSize)
+          }
+
+          Column(modifier = Modifier.weight(1f)) {
+            Text(text = headerTime, color = fieldTitleColor, fontSize = contentTextSize)
+            Text(text = contentTime, color = fieldContentColor, fontSize = contentTextSize)
+          }
+        }
+
+    // register button
+    FloatingActionButton(
+        onClick = { /*TODO*/},
+        modifier = Modifier.padding(bottom = 16.dp, end = 16.dp).align(Alignment.End),
+        containerColor = MaterialTheme.colorScheme.surfaceContainer,
+    ) {
+      Icon(
+          painter = painterResource(id = R.drawable.ticket),
+          contentDescription = "register to event button",
+          modifier = Modifier.size(32.dp),
+          tint = MaterialTheme.colorScheme.primaryContainer,
```

Comment:
Important: I think the tint could be onPrimaryContainer. Upon pulling the code, I couldn't see the ticket Icon because the color was really similar to surfaceContainer. (For elements on top of a container, use onContainerName)